libs/auth/src/java_card.ts

```diff
@@ -46,6 +64,12 @@ const OPEN_FIPS_201_AID = Buffer.from([
  */
 const DEFAULT_PIN = '000000';
 
+/**
+ * The PIN unblocking key or PUK. Typically a sensitive value but not for us given our modification
+ * to OpenFIPS201 to invalidate the card on PIN reset.
```

Comment:
nit: "invalidate the card" is a bit ambiguous, maybe more "erase the private keys"

Reply:
Settled on a combo, to be precise while also making the impact clear to someone with less context on the effect of clearing PIN-gated keys: https://github.com/votingworks/vxsuite/pull/3053/commits/f9fff7927d0faf88e5e845ea69a2879cfd3556e2

There's a more detailed comment later in the code as well: https://github.com/votingworks/vxsuite/blob/f9fff7927d0faf88e5e845ea69a2879cfd3556e2/libs/auth/src/java_card.ts#L567-L576